doc/cited.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
These make it so that sphinx does not complain about files not existing in toctrees. We link to them manually outside of toctrees.